spec/commands/get_review_recommendations_spec.rb

```diff
@@ -60,7 +60,8 @@ def perform_with_predefined_args
     end
 
     context 'with previous review requests' do
-      let!(:pull_request_1) { create(:pull_request_relation, github_user_id: github_user_id, target_user_id: other_user1.id) }
+      let!(:pull_request_1) { create(:pull_request_relation, github_user_id: other_user1.id,
+        target_user_id: github_user_id) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 64, 41 should be on its own line.